week-3/.archive/C-array-every/exercise.js

```diff
@@ -5,9 +5,11 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-var groupIsOnlyStudents; // complete this statement
+var groupIsOnlyStudents = function(students, group) {
+  return group.every(element => students.includes(element));
```

Comment:
Element can be named something else so that this is more easy to understand - maybe calling it `student` would be be better?

Reply:
Yes it makes sense.